amr-wind/equation_systems/tke/TKE.H

```diff
@@ -18,6 +19,27 @@ struct TKE : ScalarTransport
     static std::string var_name() { return "tke"; }
 };
 
+template<>
+struct TurbulenceOp<TKE>
+{
+    TurbulenceOp(turbulence::TurbulenceModel& tmodel,
+                 PDEFields& fields)
+        : m_tmodel(tmodel), m_fields(fields)
```

Comment:
clang-format

Reply:
I'm going to wait for my new computer today. This might take a day. :-)

Reply:
ok.. this is not going to happen today